media/css/firefox/family/components/modules/_privacy.scss

```diff
@@ -19,4 +19,39 @@
             background-color: f3.$green-light;
         }
     }
+
+    @media #{$mq-sm} {
+        .l-grid {
```

Comment:
**issue:** we aren't using a grid layout here

**suggestion**: move `display: grid` to mq-lg block